server/cluster.go

```diff
@@ -574,3 +574,40 @@ func (c *raftCluster) PutStore(store *metapb.Store) error {
 
 	return nil
 }
+
+func (c *raftCluster) GetMeta() (*metapb.Cluster, error) {
```

Comment:
Do we need to clone meta ? Looks like there will be race if there are multiple reader and writer.